node-tests/fixtures/builder-test/rollup/scope-override/ember-service-worker/service-worker-registration/index.js

```diff
@@ -0,0 +1,8 @@
+const SCOPE = '{{SERVICE_WORKER_SCOPE}}';
+const ROOT_URL = '{{ROOT_URL}}';
+let scopePassed = SCOPE !== 'undefined';
+let scope = scopePassed ? SCOPE : ROOT_URL;
+
+if ('serviceWorker' in navigator) {
```

Comment:
I'm worried that repeating the logic here in the fixture means that if the code changes (and breaks) the test will still pass. If we make the change of moving the logic into the service worker builder, then this fixture only has to test the scope passed in.... something like:
```
if ('serviceWorker' in navigator) {
  navigator.serviceWorker.register('{{ROOT_URL}}{{SERVICE_WORKER_FILENAME}}', { scope: '{{SERVICE_WORKER_SCOPE}}' });
};
```